frappe/email/doctype/email_account/email_account.py

```diff
@@ -375,9 +375,9 @@ def pull(now=False):
 	import frappe.tasks
 	for email_account in frappe.get_list("Email Account", filters={"enable_incoming": 1}):
 		if now:
-			frappe.tasks.pull_from_email_account(frappe.local.site, email_account.name)
+			frappe.email.doctype.email_account.email_account.pull_from_email_account(frappe.local.site, email_account.name)
 		else:
-			frappe.tasks.pull_from_email_account.delay(frappe.local.site, email_account.name)
+			frappe.email.doctype.email_account.email_account.pull_from_email_account.delay(frappe.local.site, email_account.name)
```

Comment:
Since you are already inside email_account.py, you don't need to give full path!

It should be:
`enqueue(pull_from_email_account...` and should not use .delay